README.md

```diff
@@ -47,12 +47,69 @@ require_once 'vendor/autoload.php';
 
 ### Creating the client
 
+Use the factory to create a client. Once created, you can set the various credentials to make requests.
+
 ```php
 use HelpScout\Api\ApiClientFactory;
 
 $client = ApiClientFactory::createClient();
-$client->setAccessToken('secret');
+
+// Set Auth token directly if you have it
+$client->setAccessToken('abc123');
+
+// Set Client credentials if using that grant type
+$client->useClientCredentials($appId, $appSecret);
+
+// Use legacy clientId and apiKey
+$client->useLegacyToken($clientId, $apiKey);
+
+// Use a refresh token to get a new access token
+$client->useRefreshToken($appId, $appSecret, $refreshToken);
```

Comment:
I _really_ like this approach, you've done a killer job with it!

Reply:
Thanks